.github/workflows/lib-injection.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     steps:
       - name: lib-injection test runner
         id: lib-injection-test-runner
-        uses: DataDog/system-tests/lib-injection/runner@1af3241d5b6a928199528a8cbfc5698564f5d260
+        uses: DataDog/system-tests/lib-injection/runner@4eb1ec01b61c1c7e84f219127a91d07f03b0f85e
```

Comment:
Is there any way to get the "latest", or "master"? Or would we want to explicitly always use an older version here?

(Also, would it make sense to leave a comment here explaining how to update to the latest version?)

Reply:
I was going to suggest always pointing to `master|main`. Could we do that?

Reply:
Nope. It is a SHA.

Reply:
Maybe we could use https://github.com/marketplace/actions/latest-tag ? That seems exactly what we're missing here.